checkQC/parsers/interop_parser.py

```diff
@@ -163,8 +163,7 @@ def run(self):
                 self._send_to_subscribers(("error_rate",
                                         {"lane": lane+1, 
                                          "read": read_nbr+1, 
-                                         "error_rate": error_rate,
-                                         "is_index_read":is_index_read}))
```

Comment:
I think it would make sense to keep this, but in the handler skip reads where is_index_read is true. 
That makes it extra clear that the handler is not supposed to handle index reads.